.env.example

```
@@ -5,3 +5,11 @@ CELERY_BROKER_URL=redis://localhost:6379/0
 CELERY_RESULT_BACKEND=
 CORS_ALLOWED_ORIGINS=https://app.example.com
 CSRF_TRUSTED_ORIGINS=https://app.example.com
+
+# --- Payments / BitPay ---
+PAYMENT_GATEWAY=bitpay
+BITPAY_WEBHOOK_SECRET=change-me
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The BITPAY_WEBHOOK_SECRET key should go before the PAYMENT_GATEWAY key

(UnorderedKey)